jac-uk/admin
# #1617 - eligibility report statuses

**Author checklist**


---
## What's included?
Bug - The Eligibility Issues page allows the user to choose a status to assign to each application but their choice does not get saved to the database.

1. Fix the bug 
2. Add an recommendation status filter (so the user can only see rejected applications, for example)

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Preview URL: https://jac-admin-develop--pr1619-feature-1617-eligibi-fh2lsp4z.web.app

- Load the eligibility issues page
- Set statuses for each application
- Reload the page. The chosen statuses should be visible still (i.e. they were saved to the database)
- Check the recommendation status filter works

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://user-images.githubusercontent.com/79906532/171233041-d51c0991-d2b3-45f1-8023-689c87af4c2d.mov

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
